src/ArticleLine.php

```diff
@@ -123,14 +125,36 @@ public function setSubCode($subCode)
         return $this;
     }
 
-    public function getFreeText1()
+    public function getUnits()
     {
-        return $this->freeText1;
+        return $this->units;
     }
 
-    public function setFreeText1($freeText1)
+    public function setUnits($units)
     {
-        $this->freeText1 = $freeText1;
+        $this->units = $units;
+        return $this;
+    }
+
+    public function getUnitsPriceExcl()
```

Comment:
Can you make this Money? Maybe move to a trait.

Reply:
Fixed in #142